articles/connections/references/apple-native/guides/test-siwa-connection.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Test Sign In with Apple Configuration
+description: Learn how to test the Auth0 Sign In with Apple app configuration. 
+topics:
+  - authentication
+  - connections
+  - social
+  - apple
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - add-idp
+---
+
+# Test Sign In with Apple Configuration
+
+::: note
+If you are using the Classic Universal Login flow, or embedding `Lock.js` in your application, make sure you are using `Lock.js` version 11.16 or later.
+:::
+
+1. Visit your application login page and you should see an option for Sign In with Apple:
```

Comment:
In the case of native the starting point is not a website, it is a mobile app (tricky to test)